### PR TITLE
Accu-chek Guide & Aviva Connect

### PR DESCRIPTION
**Pull request by gniezen**
See linked Trello card for details.

### CODE REVIEW

**Comment by gniezen** [comment]
[Roche Accu Chek meters](https://trello.com/c/LJRUheVG/504-roche-accu-chek-meters)

File: lib/drivers/roche/utils.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+export const APDU_TYPE = {
```
[the remark this answers is not in the document]
Comment (gniezen) [reply lib/drivers/roche/utils.js]: The stuff in utils.js is generic IEEE spec, so see that as my first attempt to abstract things out :wink: The spec allows for implementations to differ between devices (e.g., some devices provided an extended config, some don't), so maybe it's better to wait until we get another meter that uses the same spec before abstracting things out into a generic IEEE spec driver. I guess that aligns with the YAGNI principle, as I'm not currently aware of BGM manufacturers other than Roche that actually use the spec.

[the remark this answers is not in the document]
Comment (pazaan) [reply lib/drivers/roche/utils.js]: 👍

**Comment by gniezen** [comment]
@pazaan I found an issue with the `isClass` function I made. Please review changes.

**Pull request merged**